acme/agents/tf/actors.py

```diff
@@ -85,9 +85,9 @@ def observe_first(self, timestep: dm_env.TimeStep):
     if self._adder:
       self._adder.add_first(timestep)
 
-  def observe(self, action: types.NestedArray, next_timestep: dm_env.TimeStep):
+  def observe(self, action: types.NestedArray, next_timestep: dm_env.TimeStep, extras: types.NestedArray=()):
```

Comment:
> I would also like to double-check with the original author of the CL in tf/utils.py if they had reasons to do it like this. I can submit the CL in tf/utils.py myself if you don't mind.

Yes, I'm fine with it. Feel free to submit it by yourself!

Reply:
> Actually, this makes the `FeedForwardActor.observe` diverge from `Actor.observe` which has some potential negative consequences. Is there a way for you to avoid adding the parameter? E.g. could you use a custom actor which derives from FeedForwardActor?

Sorry, the `extras` parameters are not required and can be avoided. This change should not be included in the original pull request. It was my fault...